src/test/java/com/bakdata/kafka/integration/StreamsCleanUpTest.java

```diff
@@ -533,7 +529,10 @@ private KafkaStreamsApplication createComplexApplication() {
         application.setErrorTopic(errorTopicName);
         application.setBrokers(this.kafkaCluster.getBrokerList());
         application.setProductive(false);
-        application.setStreamsConfig(Map.of(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, "0"));
+        application.setStreamsConfig(Map.of(
+                StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, "0",
+                ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, "10000"
```

Comment:
Why is this necessary now?

Reply:
Because it was changed to 45s by default in Kafka 3.0. This then requires us to wait longer for clean up, thus further increasing runtime of the tests. Therefore, I just configured it to the old default (this was also the missing setting that broke most tests)